src/Microsoft.Data.Sqlite.Core/SqliteDataReader.cs

```diff
@@ -538,6 +538,18 @@ public override DataTable GetSchemaTable()
 
                         var cnt = (long)command.ExecuteScalar();
                         schemaRow[IsUnique] = cnt != 0;
+
+                        command.Parameters.Clear();
+                        command.CommandText = new StringBuilder()
+                            .AppendLine($"SELECT typeof(\"{columnName}\")")
```

Comment:
Looks like column names can contain quotes... Do a `.Replace("\"", "\"\"")` just in case.

Reply:
Done.